src/main/java/seedu/address/model/appointment/Appointment.java

```diff
@@ -28,8 +30,8 @@ public Appointment(
             AppointmentType appointmentType,
             LocalDateTime appointmentDateTime,
             Person person,
-            Sickness sickness,
-            Medicine medicine) {
+            Optional<Sickness> sickness,
```

Comment:
Would it be better to keep the regular constructor, and initialise the Optional objects inside the constructor instead?